### PR TITLE
API Update deprecations

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-framework/issues/10531

### CODE REVIEW

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
Some of the version numbers are incorrect - use `1` major versioning instead. Also apply to any other deprecation notices in this module.

File: _legacy/GraphQL/Extensions/DataObjectScaffolderExtension.php [hunk 1]
```diff
@@ -19,14 +20,19 @@
 }
 
 /**
- * @deprecated 4.8..5.0 Use silverstripe/graphql:^4 functionality.
+ * @deprecated 1.8.0 Use _graphql directory functionality instead
```
Comment (ChloeSartorelli) [inline _legacy/GraphQL/Extensions/DataObjectScaffolderExtension.php]: The message should say to use the latest version of graphql - the directory isn't the important factor here.
The message will need to be updated in some other classes below as well.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: _legacy/GraphQL/Extensions/DataObjectScaffolderExtension.php (hunk 1 above)
Comment (emteknetnz) [reply _legacy/GraphQL/Extensions/DataObjectScaffolderExtension.php]: Updated

**Review by GuySartorelli — APPROVED** [review]
MOG

**Pull request merged**